oeps/processes/oep-0001.rst

```diff
@@ -223,6 +223,9 @@ The Open edX community is given the opportunity to comment on the OEP.
 The Arbiter serves to keep the discussion on track and to bring the review
 process to a final resolution.
 
+Finally, please follow the `After Merging`_ guidelines once your change has been
```

Comment:
Why not just make a Step 4. Announcing Changes section as part of the workflow section?

Reply:
I guess my initial thought was to follow the example of the [DEPR OEP](https://docs.openedx.org/projects/openedx-proposals/en/latest/processes/oep-0021-proc-deprecation.html#removed) and put it at the bottom, as a "final" step, but I'm not wedded to that idea. I think I'll move it.